### PR TITLE
Fleury Algo added in Java

### PR DESCRIPTION
**Pull request by Riyazul555**
Closes PR #18

### CODE REVIEW

File: Fleury/Fleury.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        for (int i = 1; i <= m; i++) {
+            st = new StringTokenizer(fin.readLine());
+            int x = Integer.parseInt(st.nextToken());
+            int y = Integer.parseInt(st.nextToken());
+            G[x].add(new Pair<>(y, i));
+            G[y].add(new Pair<>(x, i));
+        }
+
+        ArrayList<Integer> e = new ArrayList<>();
+        BitSet viz = new BitSet(5 * nMax);
```
Comment (iuliagroza) [inline Fleury/Fleury.java:46]: `viz` is a frequency array for the edges of the graph, so let's initialize its size with `m`.

File: Fleury/Fleury.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    public static void main(String[] args) throws IOException {
+        BufferedReader fin = new BufferedReader(new FileReader(args[0]));
+        BufferedWriter fout = new BufferedWriter(new FileWriter(args[1]));
+
+        StringTokenizer st = new StringTokenizer(fin.readLine());
+        int n = Integer.parseInt(st.nextToken());
+        int m = Integer.parseInt(st.nextToken());
+
+        ArrayList<Pair<Integer, Integer>>[] G = new ArrayList[nMax];
+        for (int i = 0; i < nMax; i++) {
```
Comment (iuliagroza) [inline Fleury/Fleury.java:33]: ditto: use `n` instead of `nMax`

File: Fleury/Fleury.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+// Time Complexity  =  O(V + E)  where V is the number of vertices and E is the number of edges.
+// Space Complexity =  O(V + E)  where V is the number of vertices and E is the number of edges.
+
+import java.io.*;
+import java.util.*;
+
+public class EulerPath {
+
+    static final int nMax = 100005;
```
Comment (iuliagroza) [inline Fleury/Fleury.java:11]: remove

File: Fleury/Fleury.java [hunk 4]
```diff
@@ -1,15 +1,13 @@
 // This code is contributed by Riyazul555
 
-// Time Complexity  =  O(V + E)  where V is the number of vertices and E is the number of edges.
-// Space Complexity =  O(V + E)  where V is the number of vertices and E is the number of edges.
+// Time Complexity  =  O(V + E)
```
Comment (iuliagroza) [inline Fleury/Fleury.java:3]: No need to remove the legend of the symbols :)

Comment (Riyazul555) [reply Fleury/Fleury.java:3]: You mean you need those comments of what  V and E indicates
Right @iuliagroza ??

**Comment by Riyazul555** [comment]
Done @iuliagroza

**Review by iuliagroza — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**